ui/src/containers/Personas/PersonaManage/MergeForm/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { compose, withProps } from 'recompose';
+import classNames from 'classnames';
+import { withModel } from 'ui/utils/hocs';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import IdentifiersViewer from '../IdentifiersViewer';
+import AttributesViewer from '../AttributesViewer';
+import PersonaAutoComplete from '../PersonaAutoComplete';
+import MergeButton from './MergeButton';
+import styles from './styles.css';
+
+const enhance = compose(
+  withProps({ schema: 'persona' }),
+  withModel,
+  withStyles(styles)
+);
+
+const renderTarget = ({ hasTarget, mergeTargetId }) => {
+  if (!hasTarget) {
+    return <noscript />;
+  }
+
+  return (
+    <div>
+      <div className={styles.section}>
+        <h4>Identifiers</h4>
```

Comment:
Lets add some contextual help to these titles, a hover-over question mark icon with some text:

`Identifiers are linked to the actor in your statements and can be one of four different types. An indentifier can only be linked to one person, but you can move an identifier to another person by using the Reassign button in the Actions column.`

Or something like that, @Ian247 can chip in there.

Reply:
Like this wording. Suggest we add examples to the help centre documentation.

Reply:
Need to have a think about how this is displayed in the UI.